include/test/write_configuration.h

```diff
@@ -1,16 +1,17 @@
 #include <fstream>
 
-static void write_configuration(bool use_tree, int groups=1, int g_size=1) {
+static void write_configuration(bool use_tree, bool backup_in_mem=false, int groups=1, int g_size=1) {
 	// read previous configuration to get GutterTree prefix
 	// as this is system dependent and shouldn't be set by our code
 	std::ifstream in("streaming.conf");
-	std::string path_prefix;
+	std::string disk_dir = ".";
```

Comment:
don't know if this is a good idea. end user may be unpleasantly surprised to discover a giant file on their shiny new machine if they aren't fully aware of our system

Reply:
I'm saying in general. this is the same code that we use when running experiments right?

Reply:
I believe it's basically the same. Actually something we probably should do at some point is to make them literally the same.

Reply:
What do you think the solution is? We do need to use disk.

Reply:
make the user provide a `disk_dir` rather than assuming a default

Reply:
good to me